docs/src/codegen.md

```diff
@@ -5,9 +5,15 @@ title: "Test Generator"
 
 Playwright comes with the ability to generate tests out of the box and is a great way to quickly get started with testing. It will open two windows, a browser window where you interact with the website you wish to test and the Playwright Inspector window where you can record your tests, copy the tests, clear your tests as well as change the language of your tests.
 
+<video width="100%" height="100%" controls muted>
+  <source src="https://user-images.githubusercontent.com/13063165/197979804-c4fa3347-8fab-4526-a728-c1b2fbd079b4.mp4" type="video/mp4" />
+  Your browser does not support the video tag.
+</video>
```

Comment:
broken video link

Reply:
link is not broken, please check again and report if you see its not working, perhaps browser issue or just github was down at moment you checked the video